### PR TITLE
Rename site URLs

### LINKED ISSUE #960: Fix site URLs: remove numbers
The site urls are created based on the file names. Docs filenames have a prefix with a number for two reasons:
- Follow the same convention as the other weaveworks sites. Good in case we want to integrate it there some day
- Have the files sorted in our IDEs to make them easier to manage and undestand the site structure

But these numbers don't look good on the URLs so I will overwrite the urls in the frontmatter of the documents.

### PR DESCRIPTION
**Pull request by martina-if**
closes #960

### CODE REVIEW

File: site/content/community/02-roadmap.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 title: "Road Map"
 weight: 20
+url: community/road-map
 ---
 
-## Project Roadmap
+## Project Road Map
```
Comment (errordeveloper) [inline site/content/community/02-roadmap.md]: It has to be one word. It's two words we talk about roads, but the project management term is a one word.

/cc @kalbir

File: site/content/usage/22-faq.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 title: FAQ
 weight: 220
+url: usage/faq
+aliases: [ "/faq" ]
```
Comment (errordeveloper) [inline site/content/usage/22-faq.md]: nice!

File: site/content/community/02-roadmap.md (hunk 1 above)
Comment (errordeveloper) [reply site/content/community/02-roadmap.md]: In any case, we don't have to address it in this PR :)

Comment (martina-if) [reply site/content/community/02-roadmap.md]: https://dictionary.cambridge.org/dictionary/english/road-map

**Pull request merged**